src/odemis/gui/cont/acquisition.py

```diff
@@ -1395,10 +1395,15 @@ def update_acquisition_time(self):
             lvl = logging.WARN
         else:
             streams = self._tab_data_model.acquisitionStreams
-            acq_time = acqmng.estimateTime(streams)
-            acq_time = math.ceil(acq_time)  # round a bit pessimistic
-            txt = u"Estimated time is {}."
-            txt = txt.format(units.readable_time(acq_time))
+            folds = tuple(acqmng.foldStreams(self._tab_data_model.acquisitionStreams))
+            if len(streams) > len(folds):
+                txt = u"EBIC and CL streams acquired simultaneously"
+                lvl = logging.INFO
+            else:
+                acq_time = acqmng.estimateTime(streams)
+                acq_time = math.ceil(acq_time)  # round a bit pessimistic
+                txt = u"Estimated time is {}."
+                txt = txt.format(units.readable_time(acq_time))
```

Comment:
The main issue with this approach is that now if some streams are folded, it's not possible to see the estimated time. I guess the nicest way would be to have two separate texts: the info/warning message, and the time estimation. One above the other one, so that they can be both seen. If the info/warning message is empty, then the widget is not shown, to not take any space (and so the panel takes less space). Could you have a look at this?
Or do you have any idea for handling this differently?